.markdownlint.yaml

```diff
@@ -12,6 +12,9 @@ MD014: false
 # Disabling no-duplicate-headings restriction
 MD024: false
 
+# Disabling no-trailing-punctuation in headings restriction
+MD026: false
```

Comment:
not sure if we need this if my other comments are addressed 👀

Reply:
Refer to [**this**](https://github.com/fossadev/fossabot-docs/pull/20#discussion_r788770539).